### PR TITLE
chore(log): logging response message from REST call

### PR DESCRIPTION
**Pull request by mynktl**
Signed-off-by: mayank <mayank.patel@mayadata.io>

## Pull Request template

Please, go through these steps before you submit a PR. ## Remove this line

**Why is this PR required? What issue does it fix?**:

**What this PR does?**:

**Does this PR require any upgrade changes?**:

**If the changes in this PR are manually verified, list down the scenarios covered and commands you used for testing with logs:**

**Any additional information for your reviewer?**:
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [ ] Fixes #<issue number>
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: pkg/cstor/api_service.go [hunk 1]
```diff
@@ -55,12 +55,12 @@ func (p *Plugin) httpRestCall(url, reqtype string, data []byte) ([]byte, error)
 
 	respdata, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return nil, errors.Errorf("Unable to read response from maya-apiserver : %s", err.Error())
+		return nil, errors.Errorf("Unable to read response from maya-apiserver, err=%s data=%s", err.Error(), string(respdata))
```
Comment (vishnuitta) [inline pkg/cstor/api_service.go:58]: 'm not sure whether respdata can be used in all cases when ioutil.ReadAll returns error

Comment (vishnuitta) [reply pkg/cstor/api_service.go:58]: can you verify this one from docs or code?

Comment (mynktl) [reply pkg/cstor/api_service.go:58]: Yes, It can be empty or partially filled in case of error, so it won't be useful in all the cases.

Comment (vishnuitta) [reply pkg/cstor/api_service.go]: my qn is: does it crash if we try use respdata when it is nil?

Comment (mynktl) [reply pkg/cstor/api_service.go]: ok. No, it won't crash.

**Pull request merged**